src/main/java/bio/terra/workspace/service/resource/controlled/ControlledResourceService.java

```diff
@@ -41,12 +42,14 @@ public ControlledResourceService(
   public String createControlledResource(
       ControlledResource resource,
       ApiGcsBucketCreationParameters creationParameters,
-      ApiIamRole creationApiIamRole,
+      ApiPrivateResourceIamRole privateResourceIamRole,
```

Comment:
In my branch, I translated the API IAM into a WSM IAM type. I thought we wanted to avoid using the API types and convert them to WSM internal types in the controller. If you agree, then I can fix this up when I resolve conflicts on my PR.

Reply:
Yeah, this should be translated to internal types. The existing implementation is slightly off since this should be resource IAM roles instead of workspace IAM roles - I'll switch the interface + wiring up